### PR TITLE
Add hiri landing page

### PR DESCRIPTION
**Pull request by solazio**
## Done

Build hiri landing page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/engage/hiri-case-study](http://0.0.0.0:8001/engage/hiri-case-study)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check [design](https://github.com/canonical-web-and-design/www.ubuntu.com/issues/4941) compliant
- Check [brief](https://docs.google.com/document/d/1nVAMTjUPiZwxoH9ysh06fbGACcPUNVu0QhYusaOKfIA/edit#) compliant
- Check [copydoc](https://docs.google.com/document/d/1B5ek9Mfina_c9rrnuUS9kgr1kzo-jqlcqoibIvEV9oU/edit#) compliant

## Issue / Card

Fixes [#4943](https://github.com/canonical-web-and-design/www.ubuntu.com/issues/4943)

### CODE REVIEW

File: templates/engage/hiri-case-study.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1Qg3sgDrvH9w_sgJpb2fp5lyOA5R-xhGZM3aKcyuQT68/edit?ts=5cbee163#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section  class="p-strip p-strip--dark p-takeover--hiri">
+  <div class="row u-equal-height">
+    <div class="col-7  u-vertically-center">
+      <div>
+      <h2 class="u-no-margin--bottom">
```
Comment (deadlight) [inline templates/engage/hiri-case-study.html]: need to indent the contents of the div

**Comment by anasereijo** [comment]
Looks good design wise 👍

File: templates/engage/hiri-case-study.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}Private cloud security{% endblock %}
+
+{% block meta_description %}In regulated sectors, private cloud security needs to meet the highest requirements. This webinar explains how to deliver on these with an Openstack cloud.{% endblock %}
```
Comment (caldav) [inline templates/engage/hiri-case-study.html]:
```suggestion
{% block meta_description %}Business focused email client demonstrates that proprietary apps can be monetised on the Linux desktop{% endblock %}
```

File: templates/engage/hiri-case-study.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  <div class="row u-equal-height">
+    <div class="col-7  u-vertically-center">
+      <div>
+      <h2 class="u-no-margin--bottom">
+        Hiri successfully monetises their snap to dispel myths around proprietary Linux apps
+      </h2>
+      <br class="u-hide--medium u-hide--large" />
+      <p class="u-hide--medium u-hide--large">
+        <a href="#register-section" class="p-button--neutral">
+          Watch the webinar
```
Comment (caldav) [inline templates/engage/hiri-case-study.html]:
```suggestion
          Read the case study
```

**Comment by anasereijo** [comment]
The form is still looking odd

Screenshot 2019-04-27 at 10 42 55

**Comment by cristinadresch** [comment]
dup

**Comment by anthonydillon** [comment]
This branch has been superseded by Peters branch https://github.com/canonical-web-and-design/www.ubuntu.com/pull/5037

**Pull request closed without merging**